# 목표
1. 부족한 데이터 채우기

In [1]:
import urllib.request
import time
import numpy as np
import csv
import pandas as pd

from urllib.request import urlopen
from urllib.parse import quote_plus
from bs4 import BeautifulSoup
from selenium import webdriver

In [12]:
soup = []
url = []
title = []
url_href = []
title_genre = []
url_genre = []
#number = 8

1. url : url을 크롤링 한 것
2. title : url 중 class가 같은 것 수집
3. url_href : class가 같아 수집한 내용들의 href를 수집
4. number : url 개수

In [13]:
'''
# 인기 앱/게임
url_0 = 'https://play.google.com/store/apps/collection/cluster?clp=0g4cChoKFHRvcHNlbGxpbmdfZnJlZV9HQU1FEAcYAw%3D%3D:S:ANO1ljJ_Y5U&gsr=Ch_SDhwKGgoUdG9wc2VsbGluZ19mcmVlX0dBTUUQBxgD:S:ANO1ljL4b8c'
url.append(url_0)

# 인기 유로 게임
url_1 = 'https://play.google.com/store/apps/collection/cluster?clp=0g4cChoKFHRvcHNlbGxpbmdfcGFpZF9HQU1FEAcYAw%3D%3D:S:ANO1ljLtt38&gsr=Ch_SDhwKGgoUdG9wc2VsbGluZ19wYWlkX0dBTUUQBxgD:S:ANO1ljJCqyI'
url.append(url_1)

# 최고 매출 게임
url_2 = 'https://play.google.com/store/apps/collection/cluster?clp=0g4YChYKEHRvcGdyb3NzaW5nX0dBTUUQBxgD:S:ANO1ljLhYwQ&gsr=ChvSDhgKFgoQdG9wZ3Jvc3NpbmdfR0FNRRAHGAM%3D:S:ANO1ljIKta8&hl=ko'
url.append(url_2)

# 금주의 신규 추천 게임
url_3 = 'https://play.google.com/store/apps/collection/cluster?clp=SjYKKgokcHJvbW90aW9uXzMwMDA3OTFfbmV3X3JlbGVhc2VzX2dhbWVzEEoYAxIER0FNRToCCAI%3D:S:ANO1ljJYK2k&gsr=CjhKNgoqCiRwcm9tb3Rpb25fMzAwMDc5MV9uZXdfcmVsZWFzZXNfZ2FtZXMQShgDEgRHQU1FOgIIAg%3D%3D:S:ANO1ljICr10'
url.append(url_3)

# 맞춤 추천 (이건 내 아이디에 맞춰져 추천되니 장르가 많지 않을 수 있음)
url_4 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoQCAESBEdBTUUqAggCUgIIAQ%3D%3D:S:ANO1ljJlEdM&gsr=ChOiChAIARIER0FNRSoCCAJSAggB:S:ANO1ljJdubc'
url.append(url_4)

# 최근 신규 추천 게임
url_5 = 'https://play.google.com/store/apps/collection/cluster?clp=Sj8KMwotcHJvbW90aW9uXzMwMDFhMDNfbmV3X3JlbGVhc2VzX2dhbWVzX2NhdGVnb3J5EEoYAxIER0FNRToCCAI%3D:S:ANO1ljL0zR8&gsr=CkFKPwozCi1wcm9tb3Rpb25fMzAwMWEwM19uZXdfcmVsZWFzZXNfZ2FtZXNfY2F0ZWdvcnkQShgDEgRHQU1FOgIIAg%3D%3D:S:ANO1ljKuCNU&hl=ko'
url.append(url_5)

# 인스턴트 게임
url_6 = 'https://play.google.com/store/apps/collection/cluster?clp=SjAKJAoecHJvbW90aW9uXzMwMDMwOGZfaW5zdGFudGdhbWVzEEoYAxIER0FNRToCCAI%3D:S:ANO1ljLWnrU&gsr=CjJKMAokCh5wcm9tb3Rpb25fMzAwMzA4Zl9pbnN0YW50Z2FtZXMQShgDEgRHQU1FOgIIAg%3D%3D:S:ANO1ljK-g4E&hl=ko'
url.append(url_6)

# 에디터 추천 게임
url_7 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19vOVU4RlFsZzgtNBA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljJ_Ozc&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX285VThGUWxnOC00EDsYAyoCCAJSAggC:S:ANO1ljIOr3E&hl=ko'
url.append(url_7)
'''
# 기존 데이터가 부족하기 때문에 장르별로 데이터 채우기
# 맞춤 추천
url_0 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoQCAESBEdBTUUqAggCUgIIAQ%3D%3D:S:ANO1ljJlEdM&gsr=ChOiChAIARIER0FNRSoCCAJSAggB:S:ANO1ljJdubc'
url.append(url_0)

# 롤플레잉 게임
url_1 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19NalVqell2WlVRWRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljJp54o&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX01qVWp6WXZaVVFZEDsYAyoCCAJSAggC:S:ANO1ljJMRMg'
url.append(url_1)

# 아케이드 끝판왕
url_2 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY192cTRIOUdXLWNVaxA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljKNrmU&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX3ZxNEg5R1ctY1VrEDsYAyoCCAJSAggC:S:ANO1ljLbFno'
url.append(url_2)

# 두뇌 트레이닝
url_3 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19NdVpBWWNMZEFVMBA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljLnKP8&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX011WkFZY0xkQVUwEDsYAyoCCAJSAggC:S:ANO1ljKdhGk'
url.append(url_3)

# 어드밴처 게임
url_4 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19XaldDQURDQ1VEVRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljLw-Zg&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX1dqV0NBRENDVURVEDsYAyoCCAJSAggC:S:ANO1ljI4Xzk'
url.append(url_4)

# 맞춤 추천
url_5 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoQCAESBEdBTUUqAggCUgIIAQ%3D%3D:S:ANO1ljJlEdM&gsr=ChOiChAIARIER0FNRSoCCAJSAggB:S:ANO1ljJdubc'
url.append(url_5)

# 퀴즈 게임
url_6 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19YYjJsbkhqUGFaRRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljI250M&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX1hiMmxuSGpQYVpFEDsYAyoCCAJSAggC:S:ANO1ljKSWEU'
url.append(url_6)

# 시뮬레이션 게임
url_7 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19vdUJKN1JCb2JSRRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljJhsXs&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX291Qko3UkJvYlJFEDsYAyoCCAJSAggC:S:ANO1ljJaBPU'
url.append(url_7)

# 캐주얼 게임
url_8 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY190SmNKeEhpendOVRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljK6Ibs&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX3RKY0p4SGl6d05VEDsYAyoCCAJSAggC:S:ANO1ljJZT8U'
url.append(url_8)

# 뮤직 비디오 게임
url_9 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19jZ09fX3lFdUw5WRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljLCIWI&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX2NnT19feUV1TDlZEDsYAyoCCAJSAggC:S:ANO1ljLB--k'
url.append(url_9)

# 매치 3 게임
url_10 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19zLWpmTWUtc283RRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljKVKLU&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX3MtamZNZS1zbzdFEDsYAyoCCAJSAggC:S:ANO1ljKvC_c'
url.append(url_10)

# 액션 어드밴처 게임
url_11 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY184YlM2YXpwM1E1cxA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljJFpO4&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljXzhiUzZhenAzUTVzEDsYAyoCCAJSAggC:S:ANO1ljJcMjw'
url.append(url_11)

# 친구와 함께 플레이하기
url_12 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19jRnZITWtLNW1fURA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljK1SJs&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX2NGdkhNa0s1bV9REDsYAyoCCAJSAggC:S:ANO1ljLycCU'
url.append(url_12)

# 오프라인 게임
url_13 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljJipFo&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX2s1VURIUUNpbUY4EDsYAyoCCAJSAggC:S:ANO1ljJjk6U'
url.append(url_13)

# 만화스타일 게임
url_14 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY193aG1uMGJISjRkdxA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljLY7Gc&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX3dobW4wYkhKNGR3EDsYAyoCCAJSAggC:S:ANO1ljJtbVg'
url.append(url_14)

# 생활 시뮬레이션
url_15 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19rSXJleHk2WEJoVRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljJM_vw&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX2tJcmV4eTZYQmhVEDsYAyoCCAJSAggC:S:ANO1ljLtmZ0'
url.append(url_15)

# 에디터 추천 게임
url_16 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19vOVU4RlFsZzgtNBA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljJ_Ozc&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX285VThGUWxnOC00EDsYAyoCCAJSAggC:S:ANO1ljIOr3E'
url.append(url_16)

# 인스턴트 게임
url_17 = 'https://play.google.com/store/apps/collection/cluster?clp=SjAKJAoecHJvbW90aW9uXzMwMDMwOGZfaW5zdGFudGdhbWVzEEoYAxIER0FNRToCCAI%3D:S:ANO1ljLWnrU&gsr=CjJKMAokCh5wcm9tb3Rpb25fMzAwMzA4Zl9pbnN0YW50Z2FtZXMQShgDEgRHQU1FOgIIAg%3D%3D:S:ANO1ljK-g4E'
url.append(url_17)

# 논스톱 액션
url_18 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19rLW1BNU9DdXBQZxA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljKPV6w&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX2stbUE1T0N1cFBnEDsYAyoCCAJSAggC:S:ANO1ljKs43E'
url.append(url_18)

# 전략 게임
url_19 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19mRW93TVpJcTdscxA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljJMH3U&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX2ZFb3dNWklxN2xzEDsYAyoCCAJSAggC:S:ANO1ljIwjCA'
url.append(url_19)

# 스포츠 게임
url_20 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19LSkUxS2FHQ1VzNBA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljJ0IyA&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX0tKRTFLYUdDVXM0EDsYAyoCCAJSAggC:S:ANO1ljJ6_m8'
url.append(url_20)

# 탐험 어드벤처
url_21 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19vNlgzdHotVXVQVRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljLjpAI&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX282WDN0ei1VdVBVEDsYAyoCCAJSAggC:S:ANO1ljIpB7Y'
url.append(url_21)

# 핵꿀잼 모바일 게임
url_22 = 'https://play.google.com/store/apps/collection/cluster?clp=SjIKJgogcHJvbW90aW9uXzMwMDI3ZGZfdG9wX2ZyZWVfZ2FtZXMQShgDEgRHQU1FOgIIAg%3D%3D:S:ANO1ljIlWBA&gsr=CjRKMgomCiBwcm9tb3Rpb25fMzAwMjdkZl90b3BfZnJlZV9nYW1lcxBKGAMSBEdBTUU6AggC:S:ANO1ljKDJZ8'
url.append(url_22)

# 동물 시뮬레이션
url_23 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY193ZW5hQ19ITUpFTRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljKjJHg&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX3dlbmFDX0hNSkVNEDsYAyoCCAJSAggC:S:ANO1ljIOrEw'
url.append(url_23)

# 카드 게임
url_24 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY19ZYWtDeDlSSTNibxA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljI7UYE&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljX1lha0N4OVJJM2JvEDsYAyoCCAJSAggC:S:ANO1ljLZvWI'
url.append(url_24)

# 보드 게임
url_25 = 'https://play.google.com/store/apps/collection/cluster?clp=ogouCAESBEdBTUUaHAoWcmVjc190b3BpY18xcXRYcWlyUml3RRA7GAMqAggCUgIIAg%3D%3D:S:ANO1ljLAXqs&gsr=CjGiCi4IARIER0FNRRocChZyZWNzX3RvcGljXzFxdFhxaXJSaXdFEDsYAyoCCAJSAggC:S:ANO1ljKQzUA'
url.append(url_25)

# 핫이슈 인기작 모음
url_26 = 'https://play.google.com/store/apps/collection/cluster?clp=SjoKLgoocHJvbW90aW9uXzMwMDIwMWZfdG9wX3NlbGxpbmdfZnJlZV9nYW1lcxBKGAMSBEdBTUU6AggC:S:ANO1ljJRLKk&gsr=CjxKOgouCihwcm9tb3Rpb25fMzAwMjAxZl90b3Bfc2VsbGluZ19mcmVlX2dhbWVzEEoYAxIER0FNRToCCAI%3D:S:ANO1ljJ-uEE'
url.append(url_26)

# 교육 - 맞춤 추천
url_27 = 'https://play.google.com/store/apps/collection/cluster?clp=ogocCAESEEdBTUVfRURVQ0FUSU9OQUwqAggHUgIIAQ%3D%3D:S:ANO1ljLOyVE&gsr=Ch-iChwIARIQR0FNRV9FRFVDQVRJT05BTCoCCAdSAggB:S:ANO1ljKpxLU'
url.append(url_27)

# 교육 - 최고 평점 게임
url_28 = 'https://play.google.com/store/apps/collection/cluster?clp=ogocCAkSEEdBTUVfRURVQ0FUSU9OQUwqAggHUgIIAQ%3D%3D:S:ANO1ljJi0mc&gsr=Ch-iChwICRIQR0FNRV9FRFVDQVRJT05BTCoCCAdSAggB:S:ANO1ljJT9do'
url.append(url_28)

# 교육 - 프리미엄 게임
url_29 = 'https://play.google.com/store/apps/collection/cluster?clp=ogocCA0SEEdBTUVfRURVQ0FUSU9OQUwqAggHUgIIAQ%3D%3D:S:ANO1ljI3Ny8&gsr=Ch-iChwIDRIQR0FNRV9FRFVDQVRJT05BTCoCCAdSAggB:S:ANO1ljIBYQ0'
url.append(url_29)

# 교육 - 가족용 앱
url_30 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo6CAESEEdBTUVfRURVQ0FUSU9OQUwaHAoWcmVjc190b3BpY19sSE5VaWlaZV9jZxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKqodA&gsr=Cj2iCjoIARIQR0FNRV9FRFVDQVRJT05BTBocChZyZWNzX3RvcGljX2xITlVpaVplX2NnEDsYAyoCCAdSAggC:S:ANO1ljJPMXQ'
url.append(url_30)

# 교육 - 시뮬레이션 : 장르는 교육으로 들어감
url_31 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo6CAESEEdBTUVfRURVQ0FUSU9OQUwaHAoWcmVjc190b3BpY193ZW5hQ19ITUpFTRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJt4FE&gsr=Cj2iCjoIARIQR0FNRV9FRFVDQVRJT05BTBocChZyZWNzX3RvcGljX3dlbmFDX0hNSkVNEDsYAyoCCAdSAggC:S:ANO1ljIU544'
url.append(url_31)

# 교육 - 귀여운 게임
url_32 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo6CAESEEdBTUVfRURVQ0FUSU9OQUwaHAoWcmVjc190b3BpY19VR196WFQ1SEloURA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKozC8&gsr=Cj2iCjoIARIQR0FNRV9FRFVDQVRJT05BTBocChZyZWNzX3RvcGljX1VHX3pYVDVISWhREDsYAyoCCAdSAggC:S:ANO1ljK7Fs0'
url.append(url_32)

# 교육 - 학습게임
url_33 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo6CAESEEdBTUVfRURVQ0FUSU9OQUwaHAoWcmVjc190b3BpY19jVmtmMWF2a2FMVRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKNVL8&gsr=Cj2iCjoIARIQR0FNRV9FRFVDQVRJT05BTBocChZyZWNzX3RvcGljX2NWa2YxYXZrYUxVEDsYAyoCCAdSAggC:S:ANO1ljK75kU'
url.append(url_33)

# 단어 - 맞춤추천
url_34 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoVCAESCUdBTUVfV09SRCoCCAdSAggB:S:ANO1ljLO9Lw&gsr=ChiiChUIARIJR0FNRV9XT1JEKgIIB1ICCAE%3D:S:ANO1ljLTGu0'
url.append(url_34)

# 단어 - 최고 평점 게임
url_35 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoVCAkSCUdBTUVfV09SRCoCCAdSAggB:S:ANO1ljKX3es&gsr=ChiiChUICRIJR0FNRV9XT1JEKgIIB1ICCAE%3D:S:ANO1ljKXVx8'
url.append(url_35)

# 단어 - 오프라인 게임
url_36 = 'https://play.google.com/store/apps/collection/cluster?clp=ogozCAESCUdBTUVfV09SRBocChZyZWNzX3RvcGljX2s1VURIUUNpbUY4EDsYAyoCCAdSAggC:S:ANO1ljLlWho&gsr=CjaiCjMIARIJR0FNRV9XT1JEGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljKG7TQ'
url.append(url_36)

# 단어 - 프리미엄 게임
url_37 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoVCA0SCUdBTUVfV09SRCoCCAdSAggB:S:ANO1ljKL3uY&gsr=ChiiChUIDRIJR0FNRV9XT1JEKgIIB1ICCAE%3D:S:ANO1ljKWJ_4'
url.append(url_37)

# 단어 - 퀴즈 게임
url_38 = 'https://play.google.com/store/apps/collection/cluster?clp=ogozCAESCUdBTUVfV09SRBocChZyZWNzX3RvcGljXzM5Yzh6XzZxb0lrEDsYAyoCCAdSAggC:S:ANO1ljKemq8&gsr=CjaiCjMIARIJR0FNRV9XT1JEGhwKFnJlY3NfdG9waWNfMzljOHpfNnFvSWsQOxgDKgIIB1ICCAI%3D:S:ANO1ljK5v0c'
url.append(url_38)

# 단어 - 두뇌 트레이닝
url_39 = 'https://play.google.com/store/apps/collection/cluster?clp=ogozCAESCUdBTUVfV09SRBocChZyZWNzX3RvcGljX011WkFZY0xkQVUwEDsYAyoCCAdSAggC:S:ANO1ljK1dH4&gsr=CjaiCjMIARIJR0FNRV9XT1JEGhwKFnJlY3NfdG9waWNfTXVaQVljTGRBVTAQOxgDKgIIB1ICCAI%3D:S:ANO1ljJ55Ws'
url.append(url_39)

# 단어 - 퍼즐
url_40 = 'https://play.google.com/store/apps/collection/cluster?clp=ogozCAESCUdBTUVfV09SRBocChZyZWNzX3RvcGljX0F5XzNKT3BYcFJzEDsYAyoCCAdSAggC:S:ANO1ljJs7kg&gsr=CjaiCjMIARIJR0FNRV9XT1JEGhwKFnJlY3NfdG9waWNfQXlfM0pPcFhwUnMQOxgDKgIIB1ICCAI%3D:S:ANO1ljJYVTI'
url.append(url_40)

# 롤플레잉 - 맞춤 추천
url_41 = 'https://play.google.com/store/apps/collection/cluster?clp=ogodCAESEUdBTUVfUk9MRV9QTEFZSU5HKgIIB1ICCAE%3D:S:ANO1ljI17ZM&gsr=CiCiCh0IARIRR0FNRV9ST0xFX1BMQVlJTkcqAggHUgIIAQ%3D%3D:S:ANO1ljK9SNE'
url.append(url_41)

# 롤플레잉 - 프리미엄 게임
url_42 = 'https://play.google.com/store/apps/collection/cluster?clp=ogodCA0SEUdBTUVfUk9MRV9QTEFZSU5HKgIIB1ICCAE%3D:S:ANO1ljIhpCo&gsr=CiCiCh0IDRIRR0FNRV9ST0xFX1BMQVlJTkcqAggHUgIIAQ%3D%3D:S:ANO1ljL77Kg'
url.append(url_42)

# 롤플레잉 - 오프라인 게임
url_43 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo7CAESEUdBTUVfUk9MRV9QTEFZSU5HGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljKPeH8&gsr=Cj6iCjsIARIRR0FNRV9ST0xFX1BMQVlJTkcaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKvYdY'
url.append(url_43)

# 롤플레잉 - 롤플레잉 게임
url_44 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo7CAESEUdBTUVfUk9MRV9QTEFZSU5HGhwKFnJlY3NfdG9waWNfTWpVanpZdlpVUVkQOxgDKgIIB1ICCAI%3D:S:ANO1ljK-K6I&gsr=Cj6iCjsIARIRR0FNRV9ST0xFX1BMQVlJTkcaHAoWcmVjc190b3BpY19NalVqell2WlVRWRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKF-2o'
url.append(url_44)

# 롤플레잉 - 액션 어드벤처 게임
url_45 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo7CAESEUdBTUVfUk9MRV9QTEFZSU5HGhwKFnJlY3NfdG9waWNfOGJTNmF6cDNRNXMQOxgDKgIIB1ICCAI%3D:S:ANO1ljJGZMU&gsr=Cj6iCjsIARIRR0FNRV9ST0xFX1BMQVlJTkcaHAoWcmVjc190b3BpY184YlM2YXpwM1E1cxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljIM9wk'
url.append(url_45)

# 롤플레잉 - 만화스타일 게임
url_46 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo7CAESEUdBTUVfUk9MRV9QTEFZSU5HGhwKFnJlY3NfdG9waWNfd2htbjBiSEo0ZHcQOxgDKgIIB1ICCAI%3D:S:ANO1ljLqsdQ&gsr=Cj6iCjsIARIRR0FNRV9ST0xFX1BMQVlJTkcaHAoWcmVjc190b3BpY193aG1uMGJISjRkdxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJD2iI'
url.append(url_46)

# 롤플레잉 - 탐험 어드벤처
url_47 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo7CAESEUdBTUVfUk9MRV9QTEFZSU5HGhwKFnJlY3NfdG9waWNfbzZYM3R6LVV1UFUQOxgDKgIIB1ICCAI%3D:S:ANO1ljIOjOM&gsr=Cj6iCjsIARIRR0FNRV9ST0xFX1BMQVlJTkcaHAoWcmVjc190b3BpY19vNlgzdHotVXVQVRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljLcktk'
url.append(url_47)

# 보드 - 맞춤 추천
url_48 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoWCAESCkdBTUVfQk9BUkQqAggHUgIIAQ%3D%3D:S:ANO1ljKqKRI&gsr=ChmiChYIARIKR0FNRV9CT0FSRCoCCAdSAggB:S:ANO1ljItEMM'
url.append(url_48)

# 보드 - 오프라인 게임
url_49 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo0CAESCkdBTUVfQk9BUkQaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKvLAA&gsr=CjeiCjQIARIKR0FNRV9CT0FSRBocChZyZWNzX3RvcGljX2s1VURIUUNpbUY4EDsYAyoCCAdSAggC:S:ANO1ljJRiwM'
url.append(url_49)

# 보드 - 최고 평정 게임
url_50 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoWCAkSCkdBTUVfQk9BUkQqAggHUgIIAQ%3D%3D:S:ANO1ljKWhug&gsr=ChmiChYICRIKR0FNRV9CT0FSRCoCCAdSAggB:S:ANO1ljJ6jyM'
url.append(url_50)

# 보드 - 프리미엄 게임
url_51 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoWCA0SCkdBTUVfQk9BUkQqAggHUgIIAQ%3D%3D:S:ANO1ljI0wGU&gsr=ChmiChYIDRIKR0FNRV9CT0FSRCoCCAdSAggB:S:ANO1ljJ6QvI'
url.append(url_51)

# 보드 - 친구와 함께 플레이하기
url_52 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo0CAESCkdBTUVfQk9BUkQaHAoWcmVjc190b3BpY19jRnZITWtLNW1fURA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKpYDk&gsr=CjeiCjQIARIKR0FNRV9CT0FSRBocChZyZWNzX3RvcGljX2NGdkhNa0s1bV9REDsYAyoCCAdSAggC:S:ANO1ljKrH60'
url.append(url_52)

# 보드 - 전략 게임
url_53 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo0CAESCkdBTUVfQk9BUkQaHAoWcmVjc190b3BpY19mRW93TVpJcTdscxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJaWs0&gsr=CjeiCjQIARIKR0FNRV9CT0FSRBocChZyZWNzX3RvcGljX2ZFb3dNWklxN2xzEDsYAyoCCAdSAggC:S:ANO1ljI_yQM'
url.append(url_53)

# 스포츠 - 맞춤 추천
url_54 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCAESC0dBTUVfU1BPUlRTKgIIB1ICCAE%3D:S:ANO1ljKe5XI&gsr=ChqiChcIARILR0FNRV9TUE9SVFMqAggHUgIIAQ%3D%3D:S:ANO1ljK_dug'
url.append(url_54)

# 스포츠 - 프리미엄 게임
url_55 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCA0SC0dBTUVfU1BPUlRTKgIIB1ICCAE%3D:S:ANO1ljJHeOA&gsr=ChqiChcIDRILR0FNRV9TUE9SVFMqAggHUgIIAQ%3D%3D:S:ANO1ljIKJz0'
url.append(url_55)

# 스포츠 - 오프라인 게임
url_56 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfU1BPUlRTGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljKZ0KE&gsr=CjiiCjUIARILR0FNRV9TUE9SVFMaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljLaENM'
url.append(url_56)

# 스포츠 - 만화 스타일 게임
url_57 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfU1BPUlRTGhwKFnJlY3NfdG9waWNfd2htbjBiSEo0ZHcQOxgDKgIIB1ICCAI%3D:S:ANO1ljLLFJE&gsr=CjiiCjUIARILR0FNRV9TUE9SVFMaHAoWcmVjc190b3BpY193aG1uMGJISjRkdxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJwfGw'
url.append(url_57)

# 스포츠 - 캐주얼 게임
url_58 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfU1BPUlRTGhwKFnJlY3NfdG9waWNfdEpjSnhIaXp3TlUQOxgDKgIIB1ICCAI%3D:S:ANO1ljLa-H8&gsr=CjiiCjUIARILR0FNRV9TUE9SVFMaHAoWcmVjc190b3BpY190SmNKeEhpendOVRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKtiEE'
url.append(url_58)

# 스포츠 - 전략 게임
url_59 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfU1BPUlRTGhwKFnJlY3NfdG9waWNfZkVvd01aSXE3bHMQOxgDKgIIB1ICCAI%3D:S:ANO1ljLSsFM&gsr=CjiiCjUIARILR0FNRV9TUE9SVFMaHAoWcmVjc190b3BpY19mRW93TVpJcTdscxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljIpxw4'
url.append(url_59)

# 시뮬레이션 - 오프라인 게임
url_60 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo5CAESD0dBTUVfU0lNVUxBVElPThocChZyZWNzX3RvcGljX2s1VURIUUNpbUY4EDsYAyoCCAdSAggC:S:ANO1ljKZdm4&gsr=CjyiCjkIARIPR0FNRV9TSU1VTEFUSU9OGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljL0jUg'
url.append(url_60)

# 시뮬레이션 - 운전 시뮬레이션
url_61 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo5CAESD0dBTUVfU0lNVUxBVElPThocChZyZWNzX3RvcGljX0dMNEVSVTkwRXg4EDsYAyoCCAdSAggC:S:ANO1ljLS_nA&gsr=CjyiCjkIARIPR0FNRV9TSU1VTEFUSU9OGhwKFnJlY3NfdG9waWNfR0w0RVJVOTBFeDgQOxgDKgIIB1ICCAI%3D:S:ANO1ljIg7Ro'
url.append(url_61)

# 시뮬레이션 - 프리미엄 게임
url_62 = 'https://play.google.com/store/apps/collection/cluster?clp=ogobCA0SD0dBTUVfU0lNVUxBVElPTioCCAdSAggB:S:ANO1ljJPyqI&gsr=Ch6iChsIDRIPR0FNRV9TSU1VTEFUSU9OKgIIB1ICCAE%3D:S:ANO1ljILisw'
url.append(url_62)

# 시뮬레이션 - 시뮬레이션 게임
url_63 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo5CAESD0dBTUVfU0lNVUxBVElPThocChZyZWNzX3RvcGljX291Qko3UkJvYlJFEDsYAyoCCAdSAggC:S:ANO1ljK5YqU&gsr=CjyiCjkIARIPR0FNRV9TSU1VTEFUSU9OGhwKFnJlY3NfdG9waWNfb3VCSjdSQm9iUkUQOxgDKgIIB1ICCAI%3D:S:ANO1ljIPbeI'
url.append(url_63)

# 시뮬레이션 - 동물 시뮬레이션
url_64 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo5CAESD0dBTUVfU0lNVUxBVElPThocChZyZWNzX3RvcGljX3dlbmFDX0hNSkVNEDsYAyoCCAdSAggC:S:ANO1ljIQvmk&gsr=CjyiCjkIARIPR0FNRV9TSU1VTEFUSU9OGhwKFnJlY3NfdG9waWNfd2VuYUNfSE1KRU0QOxgDKgIIB1ICCAI%3D:S:ANO1ljKW-Qo'
url.append(url_64)

# 시뮬레이션 - 롤플레잉 게임
url_65 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo5CAESD0dBTUVfU0lNVUxBVElPThocChZyZWNzX3RvcGljX01qVWp6WXZaVVFZEDsYAyoCCAdSAggC:S:ANO1ljJkYhQ&gsr=CjyiCjkIARIPR0FNRV9TSU1VTEFUSU9OGhwKFnJlY3NfdG9waWNfTWpVanpZdlpVUVkQOxgDKgIIB1ICCAI%3D:S:ANO1ljK6lKI'
url.append(url_65)

# 시뮬레이션 - 클리커 게임
url_66 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo5CAESD0dBTUVfU0lNVUxBVElPThocChZyZWNzX3RvcGljX29sQkUxOXdvTE9VEDsYAyoCCAdSAggC:S:ANO1ljJtTQE&gsr=CjyiCjkIARIPR0FNRV9TSU1VTEFUSU9OGhwKFnJlY3NfdG9waWNfb2xCRTE5d29MT1UQOxgDKgIIB1ICCAI%3D:S:ANO1ljKlFOY'
url.append(url_66)

# 아케이드 - 최고 평점 게임
url_67 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCAkSC0dBTUVfQVJDQURFKgIIB1ICCAE%3D:S:ANO1ljLjoNQ&gsr=ChqiChcICRILR0FNRV9BUkNBREUqAggHUgIIAQ%3D%3D:S:ANO1ljIj13g'
url.append(url_67)

# 아케이드 - 오프라인 게임
url_68 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQVJDQURFGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljJu2wI&gsr=CjiiCjUIARILR0FNRV9BUkNBREUaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljK3jZk'
url.append(url_68)

# 아케이드 - 끝판왕
url_69 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQVJDQURFGhwKFnJlY3NfdG9waWNfdnE0SDlHVy1jVWsQOxgDKgIIB1ICCAI%3D:S:ANO1ljJh9c0&gsr=CjiiCjUIARILR0FNRV9BUkNBREUaHAoWcmVjc190b3BpY192cTRIOUdXLWNVaxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljK_uGM'
url.append(url_69)

# 아케이드 - 맞춤 추천
url_70 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCAESC0dBTUVfQVJDQURFKgIIB1ICCAE%3D:S:ANO1ljJoa6A&gsr=ChqiChcIARILR0FNRV9BUkNBREUqAggHUgIIAQ%3D%3D:S:ANO1ljJVguE'
url.append(url_70)

# 아케이드 - 캐주얼 게임
url_71 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQVJDQURFGhwKFnJlY3NfdG9waWNfdEpjSnhIaXp3TlUQOxgDKgIIB1ICCAI%3D:S:ANO1ljL1YLo&gsr=CjiiCjUIARILR0FNRV9BUkNBREUaHAoWcmVjc190b3BpY190SmNKeEhpendOVRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJ-g08'
url.append(url_71)

# 아케이드 - 만화 스타일 게임
url_72 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQVJDQURFGhwKFnJlY3NfdG9waWNfd2htbjBiSEo0ZHcQOxgDKgIIB1ICCAI%3D:S:ANO1ljKMIeE&gsr=CjiiCjUIARILR0FNRV9BUkNBREUaHAoWcmVjc190b3BpY193aG1uMGJISjRkdxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljIM_go'
url.append(url_72)

# 액션 - 롤플레잉 게임
url_73 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQUNUSU9OGhwKFnJlY3NfdG9waWNfTWpVanpZdlpVUVkQOxgDKgIIB1ICCAI%3D:S:ANO1ljIRATM&gsr=CjiiCjUIARILR0FNRV9BQ1RJT04aHAoWcmVjc190b3BpY19NalVqell2WlVRWRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKilh0'
url.append(url_73)

# 액션 - 만화 스타일 게임
url_74 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQUNUSU9OGhwKFnJlY3NfdG9waWNfd2htbjBiSEo0ZHcQOxgDKgIIB1ICCAI%3D:S:ANO1ljL_P3I&gsr=CjiiCjUIARILR0FNRV9BQ1RJT04aHAoWcmVjc190b3BpY193aG1uMGJISjRkdxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljI5ciw'
url.append(url_74)

# 액션 - 어드벤처 게임
url_75 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQUNUSU9OGhwKFnJlY3NfdG9waWNfV2pXQ0FEQ0NVRFUQOxgDKgIIB1ICCAI%3D:S:ANO1ljLW13E&gsr=CjiiCjUIARILR0FNRV9BQ1RJT04aHAoWcmVjc190b3BpY19XaldDQURDQ1VEVRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljIV6RQ'
url.append(url_75)

# 액션 - 논스톱 액션
url_76 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQUNUSU9OGhwKFnJlY3NfdG9waWNfay1tQTVPQ3VwUGcQOxgDKgIIB1ICCAI%3D:S:ANO1ljKYDZU&gsr=CjiiCjUIARILR0FNRV9BQ1RJT04aHAoWcmVjc190b3BpY19rLW1BNU9DdXBQZxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJuuFI'
url.append(url_76)

# 액션 - 프리미엄 게임
url_77 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCA0SC0dBTUVfQUNUSU9OKgIIB1ICCAE%3D:S:ANO1ljLQMHk&gsr=ChqiChcIDRILR0FNRV9BQ1RJT04qAggHUgIIAQ%3D%3D:S:ANO1ljKmbEo'
url.append(url_77)

# 액션 - 아케이드 끝판왕
url_78 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQUNUSU9OGhwKFnJlY3NfdG9waWNfdnE0SDlHVy1jVWsQOxgDKgIIB1ICCAI%3D:S:ANO1ljI2Cx4&gsr=CjiiCjUIARILR0FNRV9BQ1RJT04aHAoWcmVjc190b3BpY192cTRIOUdXLWNVaxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJr72Q'
url.append(url_78)

# 액션 - 전략 게임
url_79 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQUNUSU9OGhwKFnJlY3NfdG9waWNfZkVvd01aSXE3bHMQOxgDKgIIB1ICCAI%3D:S:ANO1ljIYIvY&gsr=CjiiCjUIARILR0FNRV9BQ1RJT04aHAoWcmVjc190b3BpY19mRW93TVpJcTdscxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKE_6A'
url.append(url_79)

# 어드벤처 - 맞춤 추천
url_80 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoaCAESDkdBTUVfQURWRU5UVVJFKgIIB1ICCAE%3D:S:ANO1ljJ1-xE&gsr=Ch2iChoIARIOR0FNRV9BRFZFTlRVUkUqAggHUgIIAQ%3D%3D:S:ANO1ljL7v5Q'
url.append(url_80)

# 어드벤처 - 프리미엄 게임
url_81 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoaCA0SDkdBTUVfQURWRU5UVVJFKgIIB1ICCAE%3D:S:ANO1ljJp5PQ&gsr=Ch2iChoIDRIOR0FNRV9BRFZFTlRVUkUqAggHUgIIAQ%3D%3D:S:ANO1ljILlrU'
url.append(url_81)

# 어드벤처 - 시뮬레이션 게임
url_82 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo4CAESDkdBTUVfQURWRU5UVVJFGhwKFnJlY3NfdG9waWNfb3VCSjdSQm9iUkUQOxgDKgIIB1ICCAI%3D:S:ANO1ljIH3Jo&gsr=CjuiCjgIARIOR0FNRV9BRFZFTlRVUkUaHAoWcmVjc190b3BpY19vdUJKN1JCb2JSRRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKEsD0'
url.append(url_82)

# 어드벤처 - 오프라인 게임
url_83 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo4CAESDkdBTUVfQURWRU5UVVJFGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljJYMAU&gsr=CjuiCjgIARIOR0FNRV9BRFZFTlRVUkUaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKxYpA'
url.append(url_83)

# 어드벤처 - 어드벤처 게임
url_84 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo4CAESDkdBTUVfQURWRU5UVVJFGhwKFnJlY3NfdG9waWNfV2pXQ0FEQ0NVRFUQOxgDKgIIB1ICCAI%3D:S:ANO1ljLVS8c&gsr=CjuiCjgIARIOR0FNRV9BRFZFTlRVUkUaHAoWcmVjc190b3BpY19XaldDQURDQ1VEVRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljLsGhk'
url.append(url_84)

# 어드벤처 - 캐주얼 게임
url_85 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo4CAESDkdBTUVfQURWRU5UVVJFGhwKFnJlY3NfdG9waWNfdEpjSnhIaXp3TlUQOxgDKgIIB1ICCAI%3D:S:ANO1ljJxJlg&gsr=CjuiCjgIARIOR0FNRV9BRFZFTlRVUkUaHAoWcmVjc190b3BpY190SmNKeEhpendOVRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljLZsgI'
url.append(url_85)

# 음악 - 맞춤 추천
url_86 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoWCAESCkdBTUVfTVVTSUMqAggHUgIIAQ%3D%3D:S:ANO1ljL3kSI&gsr=ChmiChYIARIKR0FNRV9NVVNJQyoCCAdSAggB:S:ANO1ljK-33E'
url.append(url_86)

# 음악 - 피아노 연주
url_87 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo0CAESCkdBTUVfTVVTSUMaHAoWcmVjc190b3BpY19QM3BGS0F0Wm9CMBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljI6btY&gsr=CjeiCjQIARIKR0FNRV9NVVNJQxocChZyZWNzX3RvcGljX1AzcEZLQXRab0IwEDsYAyoCCAdSAggC:S:ANO1ljI3suo'
url.append(url_87)

# 음악 - 추상적인 그래픽
url_88 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo0CAESCkdBTUVfTVVTSUMaHAoWcmVjc190b3BpY19FWURVUkJ1X0hkWRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljICZfY&gsr=CjeiCjQIARIKR0FNRV9NVVNJQxocChZyZWNzX3RvcGljX0VZRFVSQnVfSGRZEDsYAyoCCAdSAggC:S:ANO1ljLBGc0'
url.append(url_88)

# 음악 - 프리미엄 게임
url_89 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoWCA0SCkdBTUVfTVVTSUMqAggHUgIIAQ%3D%3D:S:ANO1ljIOixs&gsr=ChmiChYIDRIKR0FNRV9NVVNJQyoCCAdSAggB:S:ANO1ljJXU1A'
url.append(url_89)

# 음악 - 추상적인 영상
url_90 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo0CAESCkdBTUVfTVVTSUMaHAoWcmVjc190b3BpY19SMXhCVnlPa05mURA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljI-7oY&gsr=CjeiCjQIARIKR0FNRV9NVVNJQxocChZyZWNzX3RvcGljX1IxeEJWeU9rTmZREDsYAyoCCAdSAggC:S:ANO1ljJjkMQ'
url.append(url_90)

# 음악 - 뮤직비디오 게임
url_91 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo0CAESCkdBTUVfTVVTSUMaHAoWcmVjc190b3BpY19jZ09fX3lFdUw5WRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJnV_A&gsr=CjeiCjQIARIKR0FNRV9NVVNJQxocChZyZWNzX3RvcGljX2NnT19feUV1TDlZEDsYAyoCCAdSAggC:S:ANO1ljI1N8U'
url.append(url_91)

# 자동차 게임 - 맞춤 추천
url_92 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCAESC0dBTUVfUkFDSU5HKgIIB1ICCAE%3D:S:ANO1ljJ2BKo&gsr=ChqiChcIARILR0FNRV9SQUNJTkcqAggHUgIIAQ%3D%3D:S:ANO1ljJof8M'
url.append(url_92)

# 자동차 게임 - 최고 평점 게임
url_93 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCAkSC0dBTUVfUkFDSU5HKgIIB1ICCAE%3D:S:ANO1ljLAaPE&gsr=ChqiChcICRILR0FNRV9SQUNJTkcqAggHUgIIAQ%3D%3D:S:ANO1ljKXi5Y'
url.append(url_93)

# 자동차 게임 - 오프라인 게임
url_94 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfUkFDSU5HGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljKMO3Y&gsr=CjiiCjUIARILR0FNRV9SQUNJTkcaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJd3LQ'
url.append(url_94)

# 자동차 게임 - 레이싱 게임
url_95 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfUkFDSU5HGhwKFnJlY3NfdG9waWNfekdBR0x5eTNhVEEQOxgDKgIIB1ICCAI%3D:S:ANO1ljKlF2g&gsr=CjiiCjUIARILR0FNRV9SQUNJTkcaHAoWcmVjc190b3BpY196R0FHTHl5M2FUQRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljIh170'
url.append(url_95)

# 자동차 게임 - 프리미엄 게임
url_96 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCA0SC0dBTUVfUkFDSU5HKgIIB1ICCAE%3D:S:ANO1ljL1sr0&gsr=ChqiChcIDRILR0FNRV9SQUNJTkcqAggHUgIIAQ%3D%3D:S:ANO1ljI6u-g'
url.append(url_96)

# 자동차 게임 - 모터스포츠 게임
url_97 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfUkFDSU5HGhwKFnJlY3NfdG9waWNfTmo2T29yYUt6NVUQOxgDKgIIB1ICCAI%3D:S:ANO1ljI_akU&gsr=CjiiCjUIARILR0FNRV9SQUNJTkcaHAoWcmVjc190b3BpY19OajZPb3JhS3o1VRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJvRLo'
url.append(url_97)

# 자동차 게임 - 자동차 게임
url_98 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfUkFDSU5HGhwKFnJlY3NfdG9waWNfUTZma0JFRzFkeGsQOxgDKgIIB1ICCAI%3D:S:ANO1ljJ7r0Q&gsr=CjiiCjUIARILR0FNRV9SQUNJTkcaHAoWcmVjc190b3BpY19RNmZrQkVHMWR4axA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJjGmU'
url.append(url_98)

# 전략 - 맞춤 추천
url_99 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoZCAESDUdBTUVfU1RSQVRFR1kqAggHUgIIAQ%3D%3D:S:ANO1ljIAtTc&gsr=ChyiChkIARINR0FNRV9TVFJBVEVHWSoCCAdSAggB:S:ANO1ljIiP84'
url.append(url_99)

# 전략 - 프리미엄 게임
url_100 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoZCA0SDUdBTUVfU1RSQVRFR1kqAggHUgIIAQ%3D%3D:S:ANO1ljIGNt4&gsr=ChyiChkIDRINR0FNRV9TVFJBVEVHWSoCCAdSAggB:S:ANO1ljKaRdw'
url.append(url_100)

# 전략 - 오프라인 게임
url_101 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo3CAESDUdBTUVfU1RSQVRFR1kaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKqzc0&gsr=CjqiCjcIARINR0FNRV9TVFJBVEVHWRocChZyZWNzX3RvcGljX2s1VURIUUNpbUY4EDsYAyoCCAdSAggC:S:ANO1ljLhTkU'
url.append(url_101)

# 전략 - 캐주얼 게임
url_102 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo3CAESDUdBTUVfU1RSQVRFR1kaHAoWcmVjc190b3BpY190SmNKeEhpendOVRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljL5udM&gsr=CjqiCjcIARINR0FNRV9TVFJBVEVHWRocChZyZWNzX3RvcGljX3RKY0p4SGl6d05VEDsYAyoCCAdSAggC:S:ANO1ljJmv4Q'
url.append(url_102)

# 전략 - 에디터 추천 게임
url_103 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo3CAESDUdBTUVfU1RSQVRFR1kaHAoWcmVjc190b3BpY19vOVU4RlFsZzgtNBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKsS-k&gsr=CjqiCjcIARINR0FNRV9TVFJBVEVHWRocChZyZWNzX3RvcGljX285VThGUWxnOC00EDsYAyoCCAdSAggC:S:ANO1ljKCu8E'
url.append(url_103)

# 전략 - 전략 게임
url_104 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo3CAESDUdBTUVfU1RSQVRFR1kaHAoWcmVjc190b3BpY19mRW93TVpJcTdscxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljLhtXM&gsr=CjqiCjcIARINR0FNRV9TVFJBVEVHWRocChZyZWNzX3RvcGljX2ZFb3dNWklxN2xzEDsYAyoCCAdSAggC:S:ANO1ljKt9Xs'
url.append(url_104)

# 전략 - 어드벤처 게임
url_105 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo3CAESDUdBTUVfU1RSQVRFR1kaHAoWcmVjc190b3BpY19XaldDQURDQ1VEVRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKlMZc&gsr=CjqiCjcIARINR0FNRV9TVFJBVEVHWRocChZyZWNzX3RvcGljX1dqV0NBRENDVURVEDsYAyoCCAdSAggC:S:ANO1ljIDBdc'
url.append(url_105)

# 카드 - 맞춤 추천
url_106 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoVCAESCUdBTUVfQ0FSRCoCCAdSAggB:S:ANO1ljLaJjc&gsr=ChiiChUIARIJR0FNRV9DQVJEKgIIB1ICCAE%3D:S:ANO1ljLBzhw'
url.append(url_106)

# 카드 - 최고 평점 게임
url_107 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoVCAkSCUdBTUVfQ0FSRCoCCAdSAggB:S:ANO1ljKeA6I&gsr=ChiiChUICRIJR0FNRV9DQVJEKgIIB1ICCAE%3D:S:ANO1ljJ74LU'
url.append(url_107)

# 카드 - 프리미엄 게임
url_108 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoVCA0SCUdBTUVfQ0FSRCoCCAdSAggB:S:ANO1ljLWxgE&gsr=ChiiChUIDRIJR0FNRV9DQVJEKgIIB1ICCAE%3D:S:ANO1ljIJf_k'
url.append(url_108)

# 카드 - 카드 게임
url_109 = 'https://play.google.com/store/apps/collection/cluster?clp=ogozCAESCUdBTUVfQ0FSRBocChZyZWNzX3RvcGljX1lha0N4OVJJM2JvEDsYAyoCCAdSAggC:S:ANO1ljLAv-o&gsr=CjaiCjMIARIJR0FNRV9DQVJEGhwKFnJlY3NfdG9waWNfWWFrQ3g5UkkzYm8QOxgDKgIIB1ICCAI%3D:S:ANO1ljJXmBs'
url.append(url_109)

# 카드 - 오프라인 게임
url_110 = 'https://play.google.com/store/apps/collection/cluster?clp=ogozCAESCUdBTUVfQ0FSRBocChZyZWNzX3RvcGljX2s1VURIUUNpbUY4EDsYAyoCCAdSAggC:S:ANO1ljKDxnE&gsr=CjaiCjMIARIJR0FNRV9DQVJEGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljLSLr4'
url.append(url_110)

# 카드 - 전략 게임
url_111 = 'https://play.google.com/store/apps/collection/cluster?clp=ogozCAESCUdBTUVfQ0FSRBocChZyZWNzX3RvcGljX2ZFb3dNWklxN2xzEDsYAyoCCAdSAggC:S:ANO1ljJcqf4&gsr=CjaiCjMIARIJR0FNRV9DQVJEGhwKFnJlY3NfdG9waWNfZkVvd01aSXE3bHMQOxgDKgIIB1ICCAI%3D:S:ANO1ljJEDFU'
url.append(url_111)

# 카지노 - 인기 앱/게임
url_112 = 'https://play.google.com/store/apps/collection/cluster?clp=0g4jCiEKG3RvcHNlbGxpbmdfZnJlZV9HQU1FX0NBU0lOTxAHGAM%3D:S:ANO1ljLgBLI&gsr=CibSDiMKIQobdG9wc2VsbGluZ19mcmVlX0dBTUVfQ0FTSU5PEAcYAw%3D%3D:S:ANO1ljKRt4Q'
url.append(url_112)

# 카지노 - 최고 매출
url_113 = 'https://play.google.com/store/apps/collection/cluster?clp=0g4fCh0KF3RvcGdyb3NzaW5nX0dBTUVfQ0FTSU5PEAcYAw%3D%3D:S:ANO1ljIYPss&gsr=CiLSDh8KHQoXdG9wZ3Jvc3NpbmdfR0FNRV9DQVNJTk8QBxgD:S:ANO1ljJRAWc'
url.append(url_113)

# 캐주얼 게임 - 오프라인 게임
url_114 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQ0FTVUFMGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljKNwOM&gsr=CjiiCjUIARILR0FNRV9DQVNVQUwaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKj_00'
url.append(url_114)

# 캐주얼 게임 - 가족용 앱
url_115 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQ0FTVUFMGhwKFnJlY3NfdG9waWNfbEhOVWlpWmVfY2cQOxgDKgIIB1ICCAI%3D:S:ANO1ljLUo2k&gsr=CjiiCjUIARILR0FNRV9DQVNVQUwaHAoWcmVjc190b3BpY19sSE5VaWlaZV9jZxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljIFeQ4'
url.append(url_115)

# 캐주얼 게임 - 시뮬레이션 게임
url_116 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQ0FTVUFMGhwKFnJlY3NfdG9waWNfb3VCSjdSQm9iUkUQOxgDKgIIB1ICCAI%3D:S:ANO1ljLPwEQ&gsr=CjiiCjUIARILR0FNRV9DQVNVQUwaHAoWcmVjc190b3BpY19vdUJKN1JCb2JSRRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljLk6ns'
url.append(url_116)

# 캐주얼 게임 - 만화 스타일 게임
url_117 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQ0FTVUFMGhwKFnJlY3NfdG9waWNfd2htbjBiSEo0ZHcQOxgDKgIIB1ICCAI%3D:S:ANO1ljKsEKY&gsr=CjiiCjUIARILR0FNRV9DQVNVQUwaHAoWcmVjc190b3BpY193aG1uMGJISjRkdxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJ6DXk'
url.append(url_117)

# 캐주얼 게임 - 아케이드 끝판왕
url_118 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQ0FTVUFMGhwKFnJlY3NfdG9waWNfdnE0SDlHVy1jVWsQOxgDKgIIB1ICCAI%3D:S:ANO1ljJFeHM&gsr=CjiiCjUIARILR0FNRV9DQVNVQUwaHAoWcmVjc190b3BpY192cTRIOUdXLWNVaxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljK3-Ro'
url.append(url_118)

# 캐주얼 게임 - 퍼즐
url_119 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQ0FTVUFMGhwKFnJlY3NfdG9waWNfQXlfM0pPcFhwUnMQOxgDKgIIB1ICCAI%3D:S:ANO1ljL3FY4&gsr=CjiiCjUIARILR0FNRV9DQVNVQUwaHAoWcmVjc190b3BpY19BeV8zSk9wWHBScxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljIBMeY'
url.append(url_119)

# 캐주얼 게임 - 귀여운 게임
url_120 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfQ0FTVUFMGhwKFnJlY3NfdG9waWNfVUdfelhUNUhJaFEQOxgDKgIIB1ICCAI%3D:S:ANO1ljJFX7Q&gsr=CjiiCjUIARILR0FNRV9DQVNVQUwaHAoWcmVjc190b3BpY19VR196WFQ1SEloURA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKUULc'
url.append(url_120)

# 퀴즈 - 맞춤 추천
url_121 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCAESC0dBTUVfVFJJVklBKgIIB1ICCAE%3D:S:ANO1ljJ8Tmw&gsr=ChqiChcIARILR0FNRV9UUklWSUEqAggHUgIIAQ%3D%3D:S:ANO1ljIbeyg'
url.append(url_121)

# 퀴즈 - 퀴즈 게임
url_122 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfVFJJVklBGhwKFnJlY3NfdG9waWNfWGIybG5IalBhWkUQOxgDKgIIB1ICCAI%3D:S:ANO1ljLt2to&gsr=CjiiCjUIARILR0FNRV9UUklWSUEaHAoWcmVjc190b3BpY19YYjJsbkhqUGFaRRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKIBu8'
url.append(url_122)

# 퀴즈 - 게임 쇼 게임
url_123 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfVFJJVklBGhwKFnJlY3NfdG9waWNfQVdFWGRyclk1aEkQOxgDKgIIB1ICCAI%3D:S:ANO1ljJ0zsg&gsr=CjiiCjUIARILR0FNRV9UUklWSUEaHAoWcmVjc190b3BpY19BV0VYZHJyWTVoSRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljLnjgk'
url.append(url_123)

# 퀴즈 - 알아맞히기 게임
url_124 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfVFJJVklBGhwKFnJlY3NfdG9waWNfbUhudXBwSVFFbGMQOxgDKgIIB1ICCAI%3D:S:ANO1ljJE0xQ&gsr=CjiiCjUIARILR0FNRV9UUklWSUEaHAoWcmVjc190b3BpY19tSG51cHBJUUVsYxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljJ_edA'
url.append(url_124)

# 퀴즈 - 퀴즈 게임
url_125 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfVFJJVklBGhwKFnJlY3NfdG9waWNfMzljOHpfNnFvSWsQOxgDKgIIB1ICCAI%3D:S:ANO1ljIQEUY&gsr=CjiiCjUIARILR0FNRV9UUklWSUEaHAoWcmVjc190b3BpY18zOWM4el82cW9JaxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKe9rs'
url.append(url_125)

# 퀴즈 - 최고 평점 게임
url_126 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCAkSC0dBTUVfVFJJVklBKgIIB1ICCAE%3D:S:ANO1ljIF_os&gsr=ChqiChcICRILR0FNRV9UUklWSUEqAggHUgIIAQ%3D%3D:S:ANO1ljLvw5g'
url.append(url_126)

# 퍼즐 - 최고 평점 게임
url_127 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCAkSC0dBTUVfUFVaWkxFKgIIB1ICCAE%3D:S:ANO1ljIIXE4&gsr=ChqiChcICRILR0FNRV9QVVpaTEUqAggHUgIIAQ%3D%3D:S:ANO1ljKloNs'
url.append(url_127)

# 퍼즐 - 맞춤 추천
url_128 = 'https://play.google.com/store/apps/collection/cluster?clp=ogoXCAESC0dBTUVfUFVaWkxFKgIIB1ICCAE%3D:S:ANO1ljKwRMs&gsr=ChqiChcIARILR0FNRV9QVVpaTEUqAggHUgIIAQ%3D%3D:S:ANO1ljLd9pc'
url.append(url_128)

# 퍼즐 - 오프라인 게임
url_129 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfUFVaWkxFGhwKFnJlY3NfdG9waWNfazVVREhRQ2ltRjgQOxgDKgIIB1ICCAI%3D:S:ANO1ljKbPRw&gsr=CjiiCjUIARILR0FNRV9QVVpaTEUaHAoWcmVjc190b3BpY19rNVVESFFDaW1GOBA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKTxZI'
url.append(url_129)

# 퍼즐 - 추상적인 그래픽
url_130 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfUFVaWkxFGhwKFnJlY3NfdG9waWNfRVlEVVJCdV9IZFkQOxgDKgIIB1ICCAI%3D:S:ANO1ljKCtiw&gsr=CjiiCjUIARILR0FNRV9QVVpaTEUaHAoWcmVjc190b3BpY19FWURVUkJ1X0hkWRA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljL0NVU'
url.append(url_130)

# 퍼즐 - 논리 게임
url_131 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfUFVaWkxFGhwKFnJlY3NfdG9waWNfcjFGUzdFcF9OTXcQOxgDKgIIB1ICCAI%3D:S:ANO1ljKk3kU&gsr=CjiiCjUIARILR0FNRV9QVVpaTEUaHAoWcmVjc190b3BpY19yMUZTN0VwX05NdxA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljKUGKg'
url.append(url_131)

# 퍼즐 - 추상적인 영상
url_132 = 'https://play.google.com/store/apps/collection/cluster?clp=ogo1CAESC0dBTUVfUFVaWkxFGhwKFnJlY3NfdG9waWNfUjF4QlZ5T2tOZlEQOxgDKgIIB1ICCAI%3D:S:ANO1ljLOfJE&gsr=CjiiCjUIARILR0FNRV9QVVpaTEUaHAoWcmVjc190b3BpY19SMXhCVnlPa05mURA7GAMqAggHUgIIAg%3D%3D:S:ANO1ljK_9H4'
url.append(url_132)


In [16]:
number = len(url)

for i in range(number):
    driver = webdriver.Chrome()
    driver.get(url[i])
    driver.implicitly_wait(3) # 3초간 기다림

    cnt = 0
    while(cnt<10): # 스크롤을 10번 내림
        driver.execute_script("window.scrollTo(0, document.body.scrollHeight);")
        cnt +=1
        time.sleep(0.5)
    html = driver.page_source
    driver.quit()
    soup.append(BeautifulSoup(html, 'html.parser'))

스크롤을 내리면 새로운 내용이 추가되기 때문에 while문을 이용해 스크롤 내리는 행동을 해야함 

In [17]:
for i in range(number):
    title.append(soup[i].find_all(class_ = 'poRVub' ))
    # url_href : 우리가 필요한 href
    for i in title[i]:
        url_href.append(i.attrs['href'])

In [18]:
len(url_href)

8944

In [19]:
url_href

['/store/apps/details?id=com.supercell.brawlstars',
 '/store/apps/details?id=com.mobile.legends',
 '/store/apps/details?id=com.roblox.client',
 '/store/apps/details?id=com.supercell.clashofclans',
 '/store/apps/details?id=com.pubg.krmobile',
 '/store/apps/details?id=com.supercell.clashroyale',
 '/store/apps/details?id=com.nexon.baram',
 '/store/apps/details?id=com.maxgames.stickwarlegacy',
 '/store/apps/details?id=com.ea.game.pvz2_row',
 '/store/apps/details?id=fi.twomenandadog.zombiecatchers',
 '/store/apps/details?id=com.dts.freefireth',
 '/store/apps/details?id=com.nianticlabs.pokemongo',
 '/store/apps/details?id=com.kiloo.subwaysurf',
 '/store/apps/details?id=com.playmini.miniworld',
 '/store/apps/details?id=jp.konami.pesam',
 '/store/apps/details?id=jpark.AOS5',
 '/store/apps/details?id=com.nekki.shadowfight',
 '/store/apps/details?id=com.king.candycrushsaga',
 '/store/apps/details?id=me.pou.app',
 '/store/apps/details?id=com.orange.coloring.learn.kids',
 '/store/apps/details?id=c

8944개 자료에 중복 제거 필요
- 지금 제거하고 싶지만 지금 제거해버리면 게임 제목과 묶기 어려움

# 게임 제목 가져오기

In [20]:
game_title = []
imsi_title = []
for i in range(number):
    imsi_title.append(soup[i].find_all(class_ = 'WsMG1c nnK0zc' ))
    # url_href : 우리가 필요한 href
    for i in imsi_title[i]:
        game_title.append(i.text)

In [21]:
len(game_title)

8944

In [22]:
game_title

['브롤스타즈',
 '모바일 레전드: Bang Bang',
 'ROBLOX',
 '클래시 오브 클랜',
 '배틀그라운드',
 '클래시 로얄',
 '바람의나라: 연',
 'Stick War: Legacy',
 '플랜츠 vs. 좀비 2 Free',
 'Zombie Catchers (좀비 캐쳐) 🧟',
 'Garena Free Fire: 에볼루션',
 'Pokémon GO',
 'Subway Surfers',
 'Mini World Block Art',
 'eFootball PES 2020',
 'Anger of stick 5 : zombie  (앵거 오브 스틱 5)',
 '섀도우 파이트 2',
 '캔디크러쉬사가',
 'Pou',
 '채색 및 학습',
 '피아노 키즈 - 음악 및 노래',
 '캔디크러쉬소다',
 '팜히어로사가',
 'Standoff 2',
 'Rally Fury - 익스트림 레이싱',
 '버블 슈터 아이스 - 버블 팝',
 'Zombie Tsunami (좀비 쓰나미)',
 'Block Craft 3D: 무료 건설 게임',
 '마이토킹톰 프렌즈',
 'Hungry Shark Evolution',
 '쿠킹 매드니스 - 셰프의 레스토랑 게임',
 'Brain Out – 넌센스 마스터, 두뇌 트레이닝',
 '마이토킹톰 2',
 '헤이데이',
 '어린 이용 트럭 게임 - 집 건물 세차장',
 '앵그리버드2',
 'Dream League Soccer 2020',
 '토킹톰 골드런',
 'Hill Climb Racing',
 'Pooking - 당구대',
 '맛집 셰프',
 'Toca Kitchen 2',
 'PK XD - 우주를 탐험하고 친구들과 플레이하세요',
 '마이토킹톰 (My Talking Tom)',
 '달콤한 베이커리 왕국 - 케이크를 굽고, 장식하고, 대접해요',
 'Ninja warrior: 닌자 전사 - 모험 게임의 전설',
 'Head Ball 2',
 'Bubble Shooter',
 'Minecraft 체험판',
 'Hill Climb R

# 장르 가져오기

각 페이지 별 장르를 넣어야 하는데, 장르는 페이지 안에 있음 -> baseUrl을 두고 각 page별로 들어가서 genre들고오기

In [23]:
url_genre = []

In [24]:
baseUrl = 'https://play.google.com'
for i in range(len(url_href)):
    url2 = baseUrl + url_href[i]
    html = urllib.request.urlopen(url2).read()
    soup2 = BeautifulSoup(html, 'html.parser')
    
    title_genre = soup2.find_all(class_ = 'hrTbp R8zArc')
    url_genre.append(title_genre[1].text)


url_href_remove_duplicate에 수집된 각 url로 들어가 genre를 크롤링 해오는 과정(너무 오랜 시간이 걸림 -> 다른 해결책은?)

In [27]:
imsi_12

0
0           Action
1           Action
2        Adventure
3         Strategy
4           Action
5         Strategy
6     Role Playing
7         Strategy
8           Casual
9           Action
10          Action
11       Adventure
12          Arcade
13       Adventure
14          Sports
15          Action
16          Action
17          Casual
18          Casual
19     Educational
20           Music
21          Casual
22          Casual
23          Action
24          Racing
25          Casual
26          Arcade
27      Simulation
28          Casual
29          Arcade
...            ...
8914        Puzzle
8915        Puzzle
8916        Puzzle
8917        Puzzle
8918        Puzzle
8919        Puzzle
8920        Puzzle
8921        Puzzle
8922        Puzzle
8923        Puzzle
8924        Puzzle
8925        Puzzle
8926        Puzzle
8927        Puzzle
8928        Puzzle
8929        Puzzle
8930        Puzzle
8931        Puzzle
8932        Puzzle
8933        Puzzle
8934        Puzzle
8935        Puzzle
8936        Puzzle
8937        Puzzle
8938        Puzzle
8939        Puzzle
8940        Puzzle
8941        Puzzle
8942        Puzzle
8943        Puzzle

[8944 rows x 1 columns]

In [6]:
url_genre = pd.read_csv("url_genre")
url_href = pd.read_csv("url_href")
game_title = pd.read_csv("game_title")

In [15]:
url_genre = list(url_genre['0'])
url_href = list(url_href['0'])
game_title = list(game_title['0'])

In [16]:
len(url_genre)

8944

In [17]:
url_genre

['Action',
 'Action',
 'Adventure',
 'Strategy',
 'Action',
 'Strategy',
 'Role Playing',
 'Strategy',
 'Casual',
 'Action',
 'Action',
 'Adventure',
 'Arcade',
 'Adventure',
 'Sports',
 'Action',
 'Action',
 'Casual',
 'Casual',
 'Educational',
 'Music',
 'Casual',
 'Casual',
 'Action',
 'Racing',
 'Casual',
 'Arcade',
 'Simulation',
 'Casual',
 'Arcade',
 'Arcade',
 'Puzzle',
 'Casual',
 'Casual',
 'Educational',
 'Casual',
 'Sports',
 'Action',
 'Racing',
 'Sports',
 'Arcade',
 'Educational',
 'Adventure',
 'Casual',
 'Role Playing',
 'Adventure',
 'Sports',
 'Casual',
 'Arcade',
 'Racing',
 'Sports',
 'Music',
 'Action',
 'Arcade',
 'Casual',
 'Board',
 'Puzzle',
 'Sports',
 'Action',
 'Casual',
 'Sports',
 'Arcade',
 'Casual',
 'Action',
 'Arcade',
 'Casual',
 'Arcade',
 'Role Playing',
 'Adventure',
 'Arcade',
 'Action',
 'Strategy',
 'Educational',
 'Racing',
 'Music',
 'Adventure',
 'Casual',
 'Puzzle',
 'Action',
 'Casual',
 'Racing',
 'Racing',
 'Simulation',
 'Arcade',
 'Adv

# 분석 및 CSV 파일 만들기

게임이름, 게임 URL, 게임 장르를 분류하고, CSV 파일로 만들기

In [18]:
pd.DataFrame({'Game title' : game_title,'Url' : url_href, 'Genre' : url_genre })

Game title                                                Url  \
0                  브롤스타즈    /store/apps/details?id=com.supercell.brawlstars   
1     모바일 레전드: Bang Bang          /store/apps/details?id=com.mobile.legends   
2                 ROBLOX           /store/apps/details?id=com.roblox.client   
3              클래시 오브 클랜  /store/apps/details?id=com.supercell.clashofclans   
4                 배틀그라운드           /store/apps/details?id=com.pubg.krmobile   
...                  ...                                                ...   
8939          Love Balls      /store/apps/details?id=com.supertapx.lovedots   
8940             Merged!        /store/apps/details?id=com.gramgames.merged   
8941      Little Alchemy  /store/apps/details?id=com.sometimeswefly.litt...   
8942                Six!     /store/apps/details?id=com.gramgames.toppletap   
8943            .projekt   /store/apps/details?id=com.stampedegames.projekt   

          Genre  
0        Action  
1        Action  
2     Adventure  
3      Strategy  
4        Action  
...         ...  
8939     Puzzle  
8940     Puzzle  
8941     Puzzle  
8942     Puzzle  
8943     Puzzle  

[8944 rows x 3 columns]

여러 url을 통해 title, url, genre를 수집했습니다. 여기서 중복, 필요 없는 부분이 존재해 제거해야 합니다.
* url의 필요없는 부분 제거(/store/apps/details?id= 까지는 모두 같고, 그 뒤는 com 이거나 net .. 등임, com, net 또한 장르에 영향을 끼치지 않음)
* url을 기준 열로 잡고, 중복된 row 제거

1. url의 필요없는 부분 제거

In [19]:
url_href

['/store/apps/details?id=com.supercell.brawlstars',
 '/store/apps/details?id=com.mobile.legends',
 '/store/apps/details?id=com.roblox.client',
 '/store/apps/details?id=com.supercell.clashofclans',
 '/store/apps/details?id=com.pubg.krmobile',
 '/store/apps/details?id=com.supercell.clashroyale',
 '/store/apps/details?id=com.nexon.baram',
 '/store/apps/details?id=com.maxgames.stickwarlegacy',
 '/store/apps/details?id=com.ea.game.pvz2_row',
 '/store/apps/details?id=fi.twomenandadog.zombiecatchers',
 '/store/apps/details?id=com.dts.freefireth',
 '/store/apps/details?id=com.nianticlabs.pokemongo',
 '/store/apps/details?id=com.kiloo.subwaysurf',
 '/store/apps/details?id=com.playmini.miniworld',
 '/store/apps/details?id=jp.konami.pesam',
 '/store/apps/details?id=jpark.AOS5',
 '/store/apps/details?id=com.nekki.shadowfight',
 '/store/apps/details?id=com.king.candycrushsaga',
 '/store/apps/details?id=me.pou.app',
 '/store/apps/details?id=com.orange.coloring.learn.kids',
 '/store/apps/details?id=c

In [20]:
imsi_url_href = []
clear_url_href = []
for i in range(len(url_href)):
    imsi_url_href.append(url_href[i].replace("/store/apps/details?id=com.",""))
    clear_url_href.append(imsi_url_href[i].replace("/store/apps/details?id=",""))

In [21]:
clear_url_href

['supercell.brawlstars',
 'mobile.legends',
 'roblox.client',
 'supercell.clashofclans',
 'pubg.krmobile',
 'supercell.clashroyale',
 'nexon.baram',
 'maxgames.stickwarlegacy',
 'ea.game.pvz2_row',
 'fi.twomenandadog.zombiecatchers',
 'dts.freefireth',
 'nianticlabs.pokemongo',
 'kiloo.subwaysurf',
 'playmini.miniworld',
 'jp.konami.pesam',
 'jpark.AOS5',
 'nekki.shadowfight',
 'king.candycrushsaga',
 'me.pou.app',
 'orange.coloring.learn.kids',
 'orange.kidspiano.music.songs',
 'king.candycrushsodasaga',
 'king.farmheroessaga',
 'axlebolt.standoff2',
 'refuelgames.rally',
 'linkdesks.bubblegames.bubbleshooter',
 'net.mobigame.zombietsunami',
 'fungames.blockcraft',
 'outfit7.mytalkingtomfriends',
 'fgol.HungrySharkEvolution',
 'biglime.cookingmadness',
 'mind.quiz.brain.out',
 'outfit7.mytalkingtom2',
 'supercell.hayday',
 'gokids.transportbuilding',
 'rovio.baba',
 'firsttouchgames.dls7',
 'outfit7.talkingtomgoldrun',
 'fingersoft.hillclimb',
 'billiards.city.pool.nation.club',
 'nor

In [22]:
df = pd.DataFrame({'Game title' : game_title,'Url' : clear_url_href, 'Genre' : url_genre })


Game title                           Url      Genre
0                  브롤스타즈          supercell.brawlstars     Action
1     모바일 레전드: Bang Bang                mobile.legends     Action
2                 ROBLOX                 roblox.client  Adventure
3              클래시 오브 클랜        supercell.clashofclans   Strategy
4                 배틀그라운드                 pubg.krmobile     Action
...                  ...                           ...        ...
8939          Love Balls            supertapx.lovedots     Puzzle
8940             Merged!              gramgames.merged     Puzzle
8941      Little Alchemy  sometimeswefly.littlealchemy     Puzzle
8942                Six!           gramgames.toppletap     Puzzle
8943            .projekt         stampedegames.projekt     Puzzle

[8944 rows x 3 columns]

In [50]:
df2 = pd.read_csv("Training_set")

In [51]:
df2

Game title  \
0                                          애니팡4   
1                                     마구마구 2020   
2                                     라그나로크 오리진   
3                                  라인업: 몽타주 그리기   
4                                 Stack Colors!   
..                                          ...   
825                                  레전드 오브 룬테라   
826                                      트리비아크랙   
827                                      리틀빅시티2   
828                     스나이퍼 퓨리: 최고의 슈팅게임 - FPS   
829  진짜 기타 무료 - 기타 코드, 게임 & 음악 탭,일렉트릭 & 어쿠스틱 기타   

                                 Url         Genre  
0           sundaytoz.kakao.anipang4        Puzzle  
1                   netmarble.ma9bkr        Sports  
2         gravity.ragnarokorigin.aos  Role Playing  
3                      jhulse.lineup        Casual  
4         redforcegames.stack.colors        Casual  
..                               ...           ...  
825     riotgames.legendsofruneterra          Card  
826         etermax.preguntados.lite        Trivia  
827  gameloft.android.ANMP.GloftL2HM        Casual  
828  gameloft.android.ANMP.GloftFWHM        Action  
829                   gismart.guitar         Music  

[830 rows x 3 columns]

In [52]:
df3 = pd.concat([df,df2])

In [53]:
df3.reset_index(drop = True, inplace = True)

In [54]:
df3

Game title  \
0                                          브롤스타즈   
1                             모바일 레전드: Bang Bang   
2                                         ROBLOX   
3                                      클래시 오브 클랜   
4                                         배틀그라운드   
...                                          ...   
9769                                  레전드 오브 룬테라   
9770                                      트리비아크랙   
9771                                      리틀빅시티2   
9772                     스나이퍼 퓨리: 최고의 슈팅게임 - FPS   
9773  진짜 기타 무료 - 기타 코드, 게임 & 음악 탭,일렉트릭 & 어쿠스틱 기타   

                                  Url      Genre  
0                supercell.brawlstars     Action  
1                      mobile.legends     Action  
2                       roblox.client  Adventure  
3              supercell.clashofclans   Strategy  
4                       pubg.krmobile     Action  
...                               ...        ...  
9769     riotgames.legendsofruneterra       Card  
9770         etermax.preguntados.lite     Trivia  
9771  gameloft.android.ANMP.GloftL2HM     Casual  
9772  gameloft.android.ANMP.GloftFWHM     Action  
9773                   gismart.guitar      Music  

[9774 rows x 3 columns]

2. url을 기준으로 중복된 row 제거

In [55]:
Training_set_2 = df3.drop_duplicates('Url', keep = 'first')

In [56]:
Training_set_2.reset_index(inplace = True)
Training_set_2.drop('index', axis = 1, inplace = True)
Training_set_2

Game title                                     Url  \
0                            브롤스타즈                    supercell.brawlstars   
1               모바일 레전드: Bang Bang                          mobile.legends   
2                           ROBLOX                           roblox.client   
3                        클래시 오브 클랜                  supercell.clashofclans   
4                           배틀그라운드                           pubg.krmobile   
...                            ...                                     ...   
4329            그림자 기사: 죽음의 모험 RPG  fansipan.stickman.fight.shadow.knights   
4330                  아기돼지 길찾기 대모험                             grrr.piglet   
4331                     행성 때리기 3D               redboom.game.shootingstar   
4332           좀비 스위퍼 - 지뢰찾기 액션 퍼즐             arcgamestudio.zombiesweeper   
4333  Mow Zombies - 미소녀의 좀비 퇴치 대작전                   BlueShark.gp.MZombies   

          Genre  
0        Action  
1        Action  
2     Adventure  
3      Strategy  
4        Action  
...         ...  
4329     Action  
4330     Puzzle  
4331     Arcade  
4332     Puzzle  
4333     Casual  

[4334 rows x 3 columns]

3. csv 파일 변환

In [57]:
Training_set_2.to_csv('Training_set_2',index = False, header = False)

이상 구글 플레이스토어 크롤링 후 Training_set_2 만들기 종료

800개 데이터를 4300개까지 늘렸습니다.